pages/index.tsx

```diff
@@ -21,6 +24,19 @@ type HomePageComponent = FC & { layout?: FC<AppLayoutProps> };
 
 const HomePage: HomePageComponent = () => {
 	const { t } = useTranslation();
+	const router = useRouter();
+
+	const queryPage = router.query.page;
+	const [currentPage, setCurrentPage] = useState<TPages>('landing-page');
+
+	useEffect(() => {
+		if (queryPage === 'stats') {
+			setCurrentPage('stats-page');
```

Comment:
I think it'd be better to move this into its own page `pages/stats` rather than have conditional layout in the home page.

Reply:
Good call, I had just implemented the same logic from the devDAO branch. I didn't realized `HomeLayout` keeps the homepage header in place, which is what I was worried about losing. I just pushed a commit to create a stats page and simplify the routing.

Reply:
No problem @zhyd1997 ! Great work on the initial version, if you have any questions about the changes I made just let me know.

Reply:
Sure